### PR TITLE
Add docs for Tax#create

### PR DESCRIPTION
**Pull request by Fredar**

### CODE REVIEW

File: content/reference/endpoints/taxes.md [hunk 1]
```diff
@@ -17,8 +19,40 @@ id               | Integer | Read       | Tax's id.
...
+-----------------|---------|------------|------------
+id               | Integer | Read       | Tax's id.
+-----------------|---------|------------|------------
+name             | [Object](/reference/enums#formats)   | Read       | Tax's name, list of locales described in [enums section](/reference/enums#locales).
+notes            | String  | Read       | Tax's private notes.
+percentage       | [Decimal](/reference/enums#formats)  | Read       | Tax's percentage.
+-----------------|---------|------------|------------
+created_at       | [Time](/reference/enums#formats) | Read       | Tax's create time.
+updated_at       | [Time](/reference/enums#formats) | Read       | Tax's update time.
+start_date       | [Date](/reference/enums#formats) | Read       | Tax's start date.
```
Comment (Azdaroth) [inline content/reference/endpoints/taxes.md]: Well, for sure this is `Tax's start date', but I don't think it means anything :P I would describe it maybe as `beginning of the period when the tax is applicable`

File: content/reference/changelog.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2017-02-20
```
Comment (Azdaroth) [inline content/reference/changelog.md]: I guess we won't merge it today, tomorrow probably, 2017-02-23 makes more sense.

File: content/reference/endpoints/taxes.md (hunk 1 above)
Comment (Fredar) [inline content/reference/endpoints/taxes.md]: isint this pretty obvious? we dont explain this for bookings

Comment (Azdaroth) [inline content/reference/endpoints/taxes.md]: For bookings I think it's more obvious, for taxes arguable. @ZenCocoon @Mandaryn @LeonidMorozov @StoneFrog @thilonel thoughts?

Comment (LeonidMorozov) [inline content/reference/endpoints/taxes.md]: @Azdaroth , I think it's a good practice to document all params/attrs, even they're more then clear.

Comment (Mandaryn) [inline content/reference/endpoints/taxes.md]: @LeonidMorozov +1 for everything

Comment (ZenCocoon) [inline content/reference/endpoints/taxes.md]: I think it's a bit shady here, clearly need to describe that it because of taxable periods.

**Comment by Fredar** [comment]
@Azdaroth updated

**Pull request merged**